Packs/TwitterIOCHunter-FullDailyFeed/Integrations/TwitterIOCHunterFullDailyFeed/TwitterIOCHunterFullDailyFeed.yml

```
@@ -83,6 +83,22 @@ configuration:
   - WHITE
   required: false
   type: 15
+- additionalinfo: Select type of feed to get. Full for all, Username for filter by
+    username and Hashtag for filtered by the hashtag.
+  defaultvalue: Full
+  display: Type of Feed
+  name: typeoffeed
+  options:
+  - Full
+  - Username
+  - Hashtag
+  required: true
```

```suggestion
  - Hashtag
  required: false
```
Is it intentionally mandatory?
If not, you should change it because it has a default value.